website/content/docs/release-notes/consul-terraform-sync/v0_6_0_beta.mdx

```diff
@@ -0,0 +1,147 @@
+  Consul-Terraform-Sync beta release notes for v0.6.0
+---
+
+-> The beta release notes for v0.6.0 focus on changes and deprecations in the upcoming Consul-Terraform-Sync (CTS) v0.6.0 release. Details of these features are subject to change as the beta period commences.
+
+# New and Deprecated Features in v0.6.0
+
+**Feature Summary:**
+
+- Unify CTS UX With New CLI Start Command
+- Introduction of CLI Autocompletion
+- Addition of Features to Support HCP-Consul <EnterpriseAlert inline />
+- Addition of Features to Support TFC Agents <EnterpriseAlert inline />
+
+## Unify CTS UX With New CLI Start Command
+~> Running CTS as a daemon without using a command is deprecated in CTS 0.6.0 and will be removed at a much later date in a major release. The preferred way of running CTS is through use of the `start` command.
+
+### start
+
+The `start` command is the new and preferred way in v0.6.0 of starting CTS as a daemon. When running as a daemon, there is no default configuration to start CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:
+
+```shell-session
+$ consul-terraform-sync start -config-file=config.hcl
+```
+
+To view a list of available flags, use the `-help` or `-h` flag.
+
+##### Options
+
+The `start` command supports the following options:
+
+| Name            | Required | Type    | Description                     |Default                  |
+| --------------- | -------- | ------- | ------------------------------- | ----------------------- |
+| `-config-dir ` &nbsp; &nbsp; &nbsp;| Optional | string | A directory to load files for configuring CTS. Configuration files require an .hcl or .json file extension in order to specify their format. This option can be specified multiple times to load different directories. | none |
+| `-config-file` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;| Optional | string | A file to load for configuring CTS. Configuration file requires an .hcl or .json extension in order to specify their format. This option can be specified multiple times to load different configuration files. | none |
+| `-inspect` | Optional | boolean | Run CTS in Inspect mode to print the proposed state changes for all tasks, and then exit. No changes are applied in this mode. | false |
+| `-inspect-task` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Optional | string | Run CTS in Inspect mode to print the proposed state changes for the task, and then exit. The flag can be specified multiple times to inspect multiple tasks. No changes are applied in this mode. | none |
+| `-once` | Optional | boolean | Render templates and run tasks once. Does not start the process in long-running mode and disables buffer periods. | false |
+
+## Introduction of CLI Autocompletion
+
+In v0.6.0 the `consul-terraform-sync` command features opt-in autocompletion for flags, subcommands, and
```

Comment:
Is there a reason why we choose to do opt-in vs. enabled by default?

Reply:
I think it's because they have to install a separate script to handle the auto-completion. We just automate this installation. This is also the same UX as other products (Consul, Nomad, etc.)